frontend/src/components/explorer/gemBrowser/Metabolite.vue

```diff
@@ -70,7 +70,9 @@
             <div v-if="metabolite && metabolite.externalDbs && metabolite.externalDbs.ChEBI"
                  class="column is-3-widescreen is-2-desktop is-full-tablet has-text-centered px-2">
               <a :href="metabolite.externalDbs.ChEBI[0].url" target="_blank">
-                <img :src="`https://www.ebi.ac.uk/chebi/displayImage.do?defaultImage=true&imageIndex=0&chebiId=${metabolite.externalDbs.ChEBI[0].id.slice(6)}&dimensions=400`" />
+                <img id="chebi-img" :src="`https://www.ebi.ac.uk/chebi/displayImage.do?defaultImage=true&imageIndex=0&chebiId=${metabolite.externalDbs.ChEBI[0].id.slice(6)}&dimensions=400`" class="hoverable"/>
+                <a :href="metabolite.externalDbs.ChEBI[0].url" target="_blank"  style="display: block;">
+                  ChEBI {{metabolite.name}} </a>
```

Comment:
For people reading ltr the focus appears to be on ChEBI. I tried `ChEBI: ` and it wasn't better. How about:
```suggestion
                  {{metabolite.name}} via ChEBI</a>
```

Reply:
Tried it out and I think it looks good!